presto-main/src/test/java/io/prestosql/connector/MockConnectorFactory.java

```diff
@@ -167,9 +167,14 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
             @Override
```

Comment:
`Make table listing in MockConnector emulate Hive Connector better`

Can you explain what it means because "better" is very ambiguous?

Reply:
Sure, I've included the explanation in the commit message body.